algebird-core/src/main/scala/com/twitter/algebird/statistics/GaussianDistributionMonoid.scala

```diff
@@ -0,0 +1,28 @@
+package com.twitter.algebird.statistics
+
+import com.twitter.algebird.Monoid
+
+/**
+ * @mean - Mean
+ * @sigma - Standard deviation
+ * aka Normal distribution
+ */
+case class GaussianDistribution(mean: Double, sigma: Double)
+
+object GaussianDistribution {
+  implicit def monoid: Monoid[GaussianDistribution] = new GaussianDistributionMonoid
```

Comment:
this can be an `implicit val` to not allocate at every use (since this Monoid is a constant, and has no inputs).